packages/harfbuzz.rb

```diff
@@ -3,40 +3,47 @@
 class Harfbuzz < Package
   description 'HarfBuzz is an OpenType text shaping engine.'
   homepage 'https://www.freedesktop.org/wiki/Software/HarfBuzz/'
-  version '2.7.2-1'
+  @_ver = '2.7.4'
+  version @_ver
   compatibility 'all'
-  source_url 'https://github.com/harfbuzz/harfbuzz/releases/download/2.7.2/harfbuzz-2.7.2.tar.xz'
-  source_sha256 'b8c048d7c2964a12f2c80deb6634dfc836b603dd12bf0d0a3df1627698e220ce'
+  source_url "https://github.com/harfbuzz/harfbuzz/archive/#{@_ver}.tar.gz"
+  source_sha256 'daff8a4003ac420a8550760ed303ce33b310c8ea17b7f15b307d1969cabcebcb'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/harfbuzz-2.7.2-1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/harfbuzz-2.7.2-1-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/harfbuzz-2.7.2-1-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/harfbuzz-2.7.2-1-chromeos-x86_64.tar.xz',
+     aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/harfbuzz-2.7.4-chromeos-armv7l.tar.xz',
+      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/harfbuzz-2.7.4-chromeos-armv7l.tar.xz',
+        i686: 'https://dl.bintray.com/chromebrew/chromebrew/harfbuzz-2.7.4-chromeos-i686.tar.xz',
+      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/harfbuzz-2.7.4-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '693c5d740740dd7d0f6fa8c4174e9e88bda3a78dfab5d337c697085d06845c02',
-     armv7l: '693c5d740740dd7d0f6fa8c4174e9e88bda3a78dfab5d337c697085d06845c02',
-       i686: '7a839ce95a8cec41dda138b3545b189c558c34a6a96e1a8b3b65850e23b557e0',
-     x86_64: '4e22014938782e9ea806d73580a2d31c75acfd69d6c2302795e034545003ffc9',
+     aarch64: 'a2cc6f26453bc9077dde0d549ea92e30a8847d9c9d49ceebc681b090056d69fc',
+      armv7l: 'a2cc6f26453bc9077dde0d549ea92e30a8847d9c9d49ceebc681b090056d69fc',
+        i686: 'a98df51edf04334872597256379d041389fce9bf3b020c5700ee4d0b8771967d',
+      x86_64: 'bdcb5b9ab85752b8f8ce46b919e6739754e369fd807a5f52d85afb43ca815153',
   })
 
-  depends_on 'cairo' => ':build'
-  depends_on 'glib' => :build
-  depends_on 'gobject_introspection'
-  depends_on 'ragel' => :build
-  depends_on 'freetype_sub'
-  depends_on 'six' => :build
-  depends_on 'graphite' => :build
+ depends_on 'cairo' => ':build'
+ depends_on 'glib' => :build
+ depends_on 'gobject_introspection'
+ depends_on 'ragel' => :build
+ depends_on 'freetype_sub'
+ depends_on 'six' => :build
+ depends_on 'graphite' => :build
```

Comment:
Why was the indentation changed?

Reply:
good catch. That was a manual fix of a script error on my end... which was poorly fixed.